packages/react-topology/tsconfig.json

```diff
@@ -4,7 +4,17 @@
     "rootDir": "./src",
     "outDir": "./dist/esm",
     "declarationDir": "./dist/js",
-    "tsBuildInfoFile": "dist/esm.tsbuildinfo"
+    "tsBuildInfoFile": "dist/esm.tsbuildinfo",
+    "sourceMap": true,    
```

Comment:
I don't think we need all this extra config except for maybe `experimentalDecorators`, `suppressImplicitAnyIndexErrors` (if there are any), and `noImplicitReturns`.